LICENSE

```diff
@@ -8,8 +8,8 @@ Redistribution and use in source and binary forms, with or without modification,
 This product includes software developed by the Sileo Team.
 4. Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
 THIS SOFTWARE IS PROVIDED BY THE SILEO TEAM "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE SILEO TEAM BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-5. This software (including all forks) may not be hosted on any repo that contains advertisements, donation links, paid/sponsored software, cryptocurrency miners, or other means of monetary gain for the repo owner
+5. This software (including all forks) may not be hosted on any repo that contains advertisements, donation links, paid/sponsored software, malicious cryptocurrency miners, or other means of monetary gain for the repo owner
 6. This software may not depend on any core packages that are hosted on a repo that violates section #5
-7. This software may not be used on any paid/sponsored jailbreaks, or any jailbreak that contains advertisements, donation links, cryptocurrency miners, or any other means of monetary gain for the jailbreak developer
+7. This software may not be used on any paid/sponsored jailbreaks, or any jailbreak that contains advertisements, donation links, malicious cryptocurrency miners, or any other means of monetary gain for the jailbreak developer
```

Comment:
This clause does not need editing

Reply:
Seems like you're just changing the license to what suits you best, which is not in the interest of everyone else